tests/_runtime/test_patches.py

```diff
@@ -175,3 +187,59 @@ async def test_webbrowser_easter_egg(
 
     assert "<iframe" not in outputs[-1]
     assert "<img" in outputs[-1]
+
+
+@pytest.mark.skipif(
+    not DependencyManager.polars.has(),
+    reason="Polars is not installed",
+)
+def test_polars_write_json_patch(tmp_path: Path):
+    import sys
+
+    import polars as pl
+
+    file_path = tmp_path / "test.json"
+
+    # Fake put pyodide in sys.modules
+    sys.modules["pyodide"] = ModuleType("pyodide")
```

Comment:
Should this be in a `try`, with the unpatch in a `finally`, so that this test failing doesn't side-effect to other tests?

Reply:
i can put this in a decorator or fixture. there are a few places this exists